odoo/tools/convert.py

```diff
@@ -288,16 +288,12 @@ def _tag_report(self, rec, data_node=None, mode=None):
         for dest,f in (('name','string'),('model','model'),('report_name','name')):
             res[dest] = rec.get(f,'').encode('utf8')
             assert res[dest], "Attribute %s of report is empty !" % (f,)
-        for field,dest in (('rml','report_rml'),('file','report_rml'),('xml','report_xml'),('xsl','report_xsl'),
-                           ('attachment','attachment'),('attachment_use','attachment_use'), ('usage','usage'),
+        for field,dest in (('attachment','attachment'),('attachment_use','attachment_use'), ('usage','usage'),
```

Comment:
Maybe also fix import_xml.rng to disallow the removed attributes from <report> tags?